### PR TITLE
148-tutorials-readme

### LINKED ISSUE #148: add readme in notebooks folder
We should have a brief readme in the notebooks folder so users know how to use them.

### PR DESCRIPTION
**Pull request by elboyran**
Added all current existing links.

Fixes #148

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
 ## Tutorials
-This folder contains tutorial notebooks for DIANNA.
-A tutorial is avaiable for several combinations of data modality and explanability method.
-Click one of the links in the table to directly go to a notebook:
+This folder contains DIANNA tutorial notebooks.
 
+Tutorials are avaiable for several combinations of data modality, dataset and explanability method.
```
Comment (egpbos) [inline tutorials/README.md:4]:
```suggestion
Tutorials are available for several combinations of data modality, dataset and explanability method.
```

File: tutorials/README.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+
+<!--- to do: check if LIME is for LeafSNap (Simple Sicnetific) or more. DO we need Images row or each notebook is for 1 dataset only!? -->
+
+### Images
+|Data \ XAI|RISE|LIME|KernelSHAP|
+|:-----|:---|:---|:---|
+|_Images_|[]()|[link](lime_images.ipynb)|[]()|
+|Binary MNIST |[link](rise_mnist.ipynb)| []()| [link](kernelshap_mnist.ipynb)|
+|Simple geometric (circles and triangles) <img width="20" alt="SimpleGeometric Logo" src="https://user-images.githubusercontent.com/3244249/151539027-f2fc3fc0-282a-4993-9680-74ee28bcd360.png">|[]()| []()| [link](kernelshap_geometric_shapes.ipynb)|
+|Simple scientific (Leafsnap30) <img width="20" alt="LeafSnap30 Logo" src="https://user-images.githubusercontent.com/3244249/151539100-dbdfe0f8-485f-45d4-a249-a1f79e970066.png"> []()| []()| []()|
```
Comment (egpbos) [inline tutorials/README.md:16]:
```suggestion
<!--|Simple scientific (Leafsnap30) <img width="20" alt="LeafSnap30 Logo" src="https://user-images.githubusercontent.com/3244249/151539100-dbdfe0f8-485f-45d4-a249-a1f79e970066.png"> []()| []()| []()|-->
```

Let's comment it out as long as we don't have any notebooks yet, it's a bit weird to have an empty row in the table.

**Review by egpbos — COMMENTED** [review]
Looks good! Apart from the suggestions below, I'm wondering if we should add links to the original datasets. Maybe not necessary though, I guess people can Google them.

**Comment by egpbos** [comment]
Fixes #148

File: tutorials/README.md (hunk 2 above)
Comment (elboyran) [reply tutorials/README.md:16]: I think we have 1 notebook in progress by Yang?

**Comment by elboyran** [comment]
> Looks good! Apart from the suggestions below, I'm wondering if we should add links to the original datasets. Maybe not necessary though, I guess people can Google them.

The notebooks for generating them have links to the originals.

File: tutorials/README.md [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
-This folder contains tutorial notebooks for DIANNA.
-A tutorial is avaiable for several combinations of data modality and explanability method.
-Click one of the links in the table to directly go to a notebook:
+This folder contains DIANNA tutorial notebooks.
 
+Tutorials are available for several combinations of data modality, dataset and explanability method.
+
+Use the links in the tables below to directly go to a tutorial notebook:
+
+<!--- to do: check if LIME is for LeafSNap (Simple Sicnetific) or more. DO we need Images row or each notebook is for 1 dataset only!? -->
```
Comment (loostrum) [inline tutorials/README.md:8]: The original idea was one image dataset for each method, but there are some extra notebooks. I would only do one row for images, but multiple is also fine. LIME was only done for leafsnap

**Review by loostrum — APPROVED** [review]

File: tutorials/README.md (hunk 3 above)
Comment (elboyran) [reply tutorials/README.md:8]: I will probably redo it. Have to see how to make this PR draft again.

File: tutorials/README.md [hunk 4]
```diff
@@ -1,9 +1,18 @@
 ## Tutorials
-This folder contains tutorial notebooks for DIANNA.
-A tutorial is avaiable for several combinations of data modality and explanability method.
-Click one of the links in the table to directly go to a notebook:
+This folder contains DIANNA tutorial notebooks.
+
+Use the clicable logos below for direct access to a tutorial notebook for an explainability method and data modality/dataset:
```
Comment (egpbos) [inline tutorials/README.md:4]: Clickable

**Pull request merged**